### PR TITLE
Clarify two uses of "Raw" in CNV workflow

### PR DESCRIPTION
**Pull request by sooheelee**
I'd mistaken the two meanings of RAW in the CNV workflow and have clarified with @samuelklee and on behalf of our users.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java [hunk 1]
```diff
@@ -57,8 +57,9 @@
  * </pre>
  *
  * <p>
- *     Panel of Normal (PoN) creation uses the --transform PCOV parameter as shown.
- *     To collect raw coverage for tumor sample copy number analysis, omit this option.
+ *     To collect raw coverage, omit the --transform PCOV option.
```
Comment (samuelklee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java]: Perhaps something like: `To collect coverage in terms of integer counts (i.e., not normalized by the total number of targets), omit the --transform PCOV option.

Note: The use of "raw" in subsequent documentation may refer to either (i) coverage in terms of integer counts from this step or (ii) non-denoised coverage in general, which may be in terms of either integer counts or proportional coverage.`

And do we really use (ii) in the javadoc, or does that just happen in the tutorial? If the latter, then I would omit the note entirely. But in any case, I think you want to use the word "integer" somewhere.

**Comment by samuelklee** [comment]
Just some suggestions, thanks!

File: src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java (hunk 1 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java]: Just the tutorial, which I updated to clarify. So I'll remove this note.

**Comment by sooheelee** [comment]
I've incorporated your feedback @samuelklee. Thanks for the review.

**Comment by samuelklee** [comment]
👍

**Comment by samuelklee** [comment]
@droazen @sooheelee Let's get this doc change into the beta.

**Comment by sooheelee** [comment]
Restarted the travis tests.

**Pull request merged**